LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-color-mix-function-expected.txt

```diff
@@ -99,8 +99,14 @@ PASS e.style['color'] = "color-mix(in hwb, hwb(120deg 10% 20% / .4) 25%, hwb(30d
 PASS e.style['color'] = "color-mix(in hwb, hwb(120deg 10% 20% / .4) 30%, hwb(30deg 30% 40% / .8) 90%)" should set the property value
 PASS e.style['color'] = "color-mix(in hwb, hwb(120deg 10% 20% / .4) 12.5%, hwb(30deg 30% 40% / .8) 37.5%)" should set the property value
 PASS e.style['color'] = "color-mix(in hwb, hwb(120deg 10% 20% / .4) 0%, hwb(30deg 30% 40% / .8))" should set the property value
-PASS e.style['color'] = "color-mix(in hwb, hwb(40deg 30% 40%), hwb(60deg 30% 40%))" should set the property value
-PASS e.style['color'] = "color-mix(in hwb, hwb(60deg 30% 40%), hwb(40deg 30% 40%))" should set the property value
+FAIL e.style['color'] = "color-mix(in hwb, hwb(40deg 30% 40%), hwb(60deg 30% 40%))" should set the property value Colors do not match.
+Actual:   color-mix(in hwb, rgb(153, 127, 77), rgb(153, 153, 77))
+Expected: color-mix(in hwb, rgb(153, 128, 77), rgb(153, 153, 77)).
+Error: assert_array_approx_equals: Numeric parameters are approximately equal. property 1, expected 128 +/- 0.01, expected 128 but got 127
+FAIL e.style['color'] = "color-mix(in hwb, hwb(60deg 30% 40%), hwb(40deg 30% 40%))" should set the property value Colors do not match.
+Actual:   color-mix(in hwb, rgb(153, 153, 77), rgb(153, 127, 77))
+Expected: color-mix(in hwb, rgb(153, 153, 77), rgb(153, 128, 77)).
+Error: assert_array_approx_equals: Numeric parameters are approximately equal. property 4, expected 128 +/- 0.01, expected 128 but got 127
```

Comment:
What should I think about all the new 127 vs. 128 failures in this file?

Reply:
We don't round 127.5 correctly https://bugs.webkit.org/show_bug.cgi?id=256888

Reply:
This comes down to a small shift when doing the math where we get 0.500000 as the green value that when scaled to be out of 255 becomes 127.499985 and gets rounded down.

We currently use use `round`, but I think the CSS spec wants us to `ceil`:

> The percentage reference range of the color channels comes from the historical fact that many graphics engines stored the color channels internally as a single byte, which can hold integers between 0 and 255. Implementations should honor the precision of the channel as authored or calculated wherever possible. If this is not possible, the channel should be [rounded towards +∞](https://drafts.csswg.org/css-values-4/#combine-integers).

The linked text at the end unfortunately kind of contradicts that, saying only rounding towards +∞ when exactly 0.5.

Will be following up on this, but probably the test should be updated to allow +/- 1, not +/- 0.01 since these are integers.